app/models/item.rb

```diff
@@ -6,9 +6,12 @@ class Item < ActiveRecord::Base
 
   has_attached_file :image, styles: {
       thumb: '100x100>',
-      square: '200x200#',
+      square: '200x200',
       medium: '300x300>'
-    }
+    },
+    :storage => :s3,
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.